_bibliography/papers.bib

```diff
@@ -136,6 +136,9 @@ @inproceedings{peng2023reproducing
     booktitle=ASRU,
     year={2023},
     publisher={IEEE},
+    code={https://github.com/espnet/espnet},
+    blog={https://www.wavlab.org/activities/2023/foundations/},
+    img={../assets/img/blog/owsm_pipeline.png}
```

Comment:
I think you might forget to upload the images?

Btw, it might be better to create some directory structure for this activity (otherwise we might have a lot of images in one folder